src/go/cmd/content.go

```diff
@@ -28,6 +28,8 @@ func (c content) isEmpty() bool {
 
 // adds the specified const declaration to the exports list
 func (c *content) addConst(pkg pkg, g *ast.GenDecl) {
+	// skip adding the const type in the token list
+	const skip = "skip"
```

Comment:
Should make this exported so that the cmd can use it.

Reply:
Not sure I understand what you mean here? Should I take it out of the method so that the token builder can reference the same value? Or integrate it into the command somehow (O.o)?

Reply:
> Should I take it out of the method so that the token builder can reference the same value?

Exactly this.

Reply:
done